test/modules/utilities.test.js

```diff
@@ -315,7 +316,28 @@ describe('Utilities module', () => {
         assert.equal(result.length, 9);
     });
 
-    // enable after() step after above .skip()ed tests are fixed
+    it('check normalizeHex', () => {
+        const myNormalizedHex = Utilities.normalizeHex('123456789');
+        assert.equal(myNormalizedHex.indexOf('0x'), 0, 'myNormalizedHex doesnt start with 0x');
+        assert.equal(myNormalizedHex.length - 2, 9);
+    });
+
+    it('check getExternalIp', async () => {
```

Comment:
This test depends on external service and is error prone. Should not be in the tests.

Reply:
ok, I can skip or delete it